neurom/stats.py

```diff
@@ -176,3 +176,26 @@ def compare_two(data1, data2, test=StatTests.ks):
     Stats = namedtuple('Stats', ['dist', 'pvalue'])
 
     return Stats(*results)
+
+
+def total_score(paired_dats, p=2, test=StatTests.ks):
```

Comment:
That doesn't really answer my question. But if `p` can be independently varied for the same input data, then this should be tested.